### PR TITLE
New phone (no image)

### PR DESCRIPTION
**Pull request by Bart-Spons**

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -510,3 +510,57 @@ function bfsShortestPath(graph, startNode) {
...
 }
+
+// switch in the phone between tabs
+phoneNav.querySelectorAll("button").forEach((button) => {
+    button.addEventListener("click", () => {
+        phoneNav.querySelectorAll("button").forEach((button) => {
+            button.classList.remove("active");
+        });
+        button.classList.add("active");
+        phoneTabs.forEach((tab) => {
```
Comment (joppekoops) [inline js/app.js]: phoneTabs is not defined

File: styles/style.css [hunk 2]
```diff
@@ -640,7 +658,8 @@ FAKEBOOK STYLES
 }
 
 .friends {
-
+    margin-left: -0.5em;
```
Comment (joppekoops) [inline styles/style.css]: Makes it overflow its container

File: styles/style.css [hunk 3]
```diff
@@ -572,23 +573,40 @@ FAKEBOOK STYLES
...
+        --background-color: #ffffff 20%;
+        
+    }
+}
+
+.friends, .liked, .profile {
+    display: none;
+}
+
+section#home {
```
Comment (joppekoops) [inline styles/style.css]: No ID's in the CSS 😡

File: styles/style.css [hunk 4]
```diff
@@ -691,4 +710,46 @@ FAKEBOOK STYLES
 
 .delete-button {
     background-color: #d70000;
+}
+
+
+
+#selectedProfile {
```
Comment (joppekoops) [inline styles/style.css]: This belongs at the account section of the file

**Review by joppekoops — CHANGES_REQUESTED** [review]

File: js/app.js (hunk 1 above)
Comment (Bart-Spons) [reply js/app.js]: It is now

File: styles/style.css (hunk 2 above)
Comment (Bart-Spons) [reply styles/style.css]: removed it both

File: styles/style.css (hunk 3 above)
Comment (Bart-Spons) [reply styles/style.css]: sorry boss

File: styles/style.css (hunk 4 above)
Comment (Bart-Spons) [reply styles/style.css]: Didn't exist so I made one

**Review by joppekoops — APPROVED** [review]

**Pull request merged**